---a/net/minecraft/entity/EntityFlying.java
+++b/net/minecraft/entity/EntityFlying.java
@@ -5,43 +5,38 @@
 import net.minecraft.util.MathHelper;
 import net.minecraft.world.World;
 
-public abstract class EntityFlying extends EntityLiving
-{
+public abstract class EntityFlying extends EntityLiving {
+
     private static final String __OBFID = "CL_00001545";
 
-    public EntityFlying(World worldIn)
-    {
+    public EntityFlying(World worldIn) {
         super(worldIn);
     }
 
-    public void fall(float distance, float damageMultiplier) {}
+    public void fall(float distance, float damageMultiplier) {
+    }
 
-    protected void func_180433_a(double p_180433_1_, boolean p_180433_3_, Block p_180433_4_, BlockPos p_180433_5_) {}
+    protected void func_180433_a(double p_180433_1_, boolean p_180433_3_, Block p_180433_4_, BlockPos p_180433_5_) {
+    }
 
-    public void moveEntityWithHeading(float p_70612_1_, float p_70612_2_)
-    {
-        if (this.isInWater())
-        {
+    public void moveEntityWithHeading(float p_70612_1_, float p_70612_2_) {
+        if (this.isInWater()) {
             this.moveFlying(p_70612_1_, p_70612_2_, 0.02F);
             this.moveEntity(this.motionX, this.motionY, this.motionZ);
             this.motionX *= 0.800000011920929D;
             this.motionY *= 0.800000011920929D;
             this.motionZ *= 0.800000011920929D;
-        }
-        else if (this.isInLava())
-        {
+        } else if (this.isInLava()) {
             this.moveFlying(p_70612_1_, p_70612_2_, 0.02F);
             this.moveEntity(this.motionX, this.motionY, this.motionZ);
             this.motionX *= 0.5D;
             this.motionY *= 0.5D;
             this.motionZ *= 0.5D;
-        }
-        else
-        {
+        } else {
             float f2 = 0.91F;
 
-            if (this.onGround)
-            {
+            if (this.onGround) {
                 f2 = this.worldObj.getBlockState(new BlockPos(MathHelper.floor_double(this.posX), MathHelper.floor_double(this.getEntityBoundingBox().minY) - 1, MathHelper.floor_double(this.posZ))).getBlock().slipperiness * 0.91F;
             }
 
@@ -49,14 +49,12 @@
             this.moveFlying(p_70612_1_, p_70612_2_, this.onGround ? 0.1F * f3 : 0.02F);
             f2 = 0.91F;
 
-            if (this.onGround)
-            {
+            if (this.onGround) {
                 f2 = this.worldObj.getBlockState(new BlockPos(MathHelper.floor_double(this.posX), MathHelper.floor_double(this.getEntityBoundingBox().minY) - 1, MathHelper.floor_double(this.posZ))).getBlock().slipperiness * 0.91F;
             }
 
             this.moveEntity(this.motionX, this.motionY, this.motionZ);
-            this.motionX *= (double)f2;
-            this.motionY *= (double)f2;
-            this.motionZ *= (double)f2;
+            this.motionX *= (double) f2;
+            this.motionY *= (double) f2;
+            this.motionZ *= (double) f2;
         }
 
         this.prevLimbSwingAmount = this.limbSwingAmount;
@@ -65,8 +65,7 @@
         double d0 = this.posZ - this.prevPosZ;
         float f4 = MathHelper.sqrt_double(d1 * d1 + d0 * d0) * 4.0F;
 
-        if (f4 > 1.0F)
-        {
+        if (f4 > 1.0F) {
             f4 = 1.0F;
         }
 
@@ -74,9 +74,7 @@
         this.limbSwing += this.limbSwingAmount;
     }
 
-    public boolean isOnLadder()
-    {
+    public boolean isOnLadder() {
         return false;
     }
-}
+}
